virttest/remote.py

```diff
@@ -119,7 +119,7 @@ def wait_for_ssh_login_to_migrate(client, host, port, username, password, prompt
     while time.time() < end_time:
         try:
             return ssh_login_to_migrate(client, host, port, username, password, prompt,
-                                        linesep, log_filename, log_function,
```

Comment:
Could we instead add the login function argument in `ssh_login_to_migrate`?
```diff
def ssh_login_to_migrate(client, host, port, username, password, prompt, linesep="\n",
-                         log_filename=None, timeout=10, interface=None, identity_file=None,
+                         log_filename=None, log_filename=None, timeout=10, interface=None, identity_file=None,
                         status_test_command="echo $?", verbose=False, bind_ip=None,
...
    if client != "ssh":
        return remote_login(client, host, port, username, password, prompt,
-                            linesep, log_filename, None, timeout,
+                            linesep, log_filename, log_function, timeout,
                            interface, identity_file, status_test_command,
```
? As you can guess from the name this function is new and temporary thus its API change should not matter too much and the change I propose is in the direction of this migration.

Reply:
Sure, thanks for making the step forward!